docs/en/supported_models/supported_models.md

```diff
@@ -39,36 +39,37 @@ The TurboMind engine doesn't support window attention. Therefore, for models tha
-|     InternLM2.5     |     7B      |    Yes    |   No    |  -   |
-|      Baichuan2      |  7B - 13B   |    Yes    |   No    | Yes  |
-|      ChatGLM2       |     6B      |    Yes    |   No    |  No  |
-|       Falcon        |  7B - 180B  |    Yes    |   No    |  No  |
-|         YI          |  6B - 34B   |    Yes    |   No    |  No  |
-|       Mistral       |     7B      |    Yes    |   No    |  No  |
-|       Mixtral       |    8x7B     |    Yes    |   No    |  No  |
-|        QWen         | 1.8B - 72B  |    Yes    |   No    |  No  |
-|       QWen1.5       | 0.5B - 110B |    Yes    |   No    |  No  |
-|     QWen1.5-MoE     |    A2.7B    |    Yes    |   No    |  No  |
-|        QWen2        | 0.5B - 72B  |    Yes    |   No    |  No  |
-|    DeepSeek-MoE     |     16B     |    Yes    |   No    |  No  |
-|     DeepSeek-V2     |  16B, 236B  |    Yes    |   No    |  No  |
-|        Gemma        |    2B-7B    |    Yes    |   No    |  No  |
-|        Dbrx         |    132B     |    Yes    |   No    |  No  |
-|     StarCoder2      |   3B-15B    |    Yes    |   No    |  No  |
-|     Phi-3-mini      |    3.8B     |    Yes    |   No    |  No  |
-|    Phi-3-vision     |    4.2B     |    Yes    |   No    |  No  |
-|     CogVLM-Chat     |     17B     |    Yes    |   No    |  No  |
-|    CogVLM2-Chat     |     19B     |    Yes    |   No    |  No  |
-|   LLaVA(1.5,1.6)    |   7B-34B    |    Yes    |   No    |  No  |
-| InternVL-Chat(v1.5) |   2B-26B    |    Yes    |   No    |  No  |
-|      InternVL2      |   1B-40B    |    Yes    |   No    |  No  |
-|       Gemma2        |   9B-27B    |    Yes    |   No    |  No  |
-|        GLM4         |     9B      |    Yes    |   No    |  No  |
-|      CodeGeeX4      |     9B      |    Yes    |   No    |  No  |
+|        Model        |    Size     | FP16/BF16 | KV INT8 | W8A8 | W4A16 |
+| :-----------------: | :---------: | :-------: | :-----: | :--: | :---: |
+|        Llama        |  7B - 65B   |    Yes    |   No    | Yes  |  Yes  |
+|       Llama2        |  7B - 70B   |    Yes    |   No    | Yes  |  Yes  |
+|       Llama3        |   8B, 70B   |    Yes    |   No    | Yes  |  Yes  |
+|      InternLM       |  7B - 20B   |    Yes    |   No    | Yes  |   -   |
+|      InternLM2      |  7B - 20B   |    Yes    |   No    | Yes  |  Yes  |
+|     InternLM2.5     |     7B      |    Yes    |   No    | Yes  |  Yes  |
+|      Baichuan2      |     7B      |    Yes    |   No    | Yes  |  Yes  |
+|      Baichuan2      |     13B     |    Yes    |   No    |  No  |  No   |
+|      ChatGLM2       |     6B      |    Yes    |   No    |  No  |  No   |
+|       Falcon        |  7B - 180B  |    Yes    |   No    |  No  |  No   |
+|         YI          |  6B - 34B   |    Yes    |   No    |  No  |  Yes  |
+|       Mistral       |     7B      |    Yes    |   No    |  No  |  No   |
+|       Mixtral       |    8x7B     |    Yes    |   No    |  No  |  No   |
+|        QWen         | 1.8B - 72B  |    Yes    |   No    |  No  |  Yes  |
```

Comment:
`/mnt/140/Qwen/qwen-7b-chat-4bit` failed with:
```
  File "/projects/lmdeploy/lmdeploy/pytorch/weight_loader/model_weight_loader.py", line 111, in _load_shard_for_key
    raise RuntimeError(f'Unknown weight: {key}.')
RuntimeError: Unknown weight: transformer.h.0.attn.c_attn.weight.
```

Reply:
Not a valid model